src/muse/utilities.py

```diff
@@ -200,14 +198,20 @@ def broadcast_techs(
     This function broadcast the first representation to the shape and coordinates
     of the second.
 
+    Note: this is not necessarily limited to `technology` datasets. For
```

Comment:
Since this function is not limited to technology datasets, it should be renamed to something different as well as well as use variable names that do not indicate the type of data. Maybe `broadcast_data` for the function name, and just `data` or `dataset` instead of `technologies`. The docstring and comments will need to be updated accordingly.